src/Microsoft/Windows/ComputeVirtualization/HostNetworkingService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ComponentModel;
+using Microsoft.Windows.ComputeVirtualization.Schema;
+
+namespace Microsoft.Windows.ComputeVirtualization
+{
+    public class HostNetworkingService
+    {
+        /// <summary>
+        /// Helper function to call into HNS to update/query a single network.
+        /// </summary>
+        /// <param name="method">The request method type</param>
+        /// <param name="path">The path of the request</param>
+        /// <param name="request">The request data</param>
+        /// <returns>The queried network.</returns>
+        public static HNSNetwork HNSNetworkRequest(string method, string path, string request)
```

Comment:
Do you want to expose this in a more concrete way? That is, should we take the opportunity to provide wrappers for this for the different things it can do? It's not a very clear interface to require callers to specify the method, path, and request as strings.